src/components/layout/total.js

```diff
@@ -22,24 +29,89 @@ const Render = ({ total }) => {
                     {total} ETC
                 </Col>
             </Row>
+            <Row>
+                <Col xs={12} style={fiatSubtitle}>
+                    <span style={valueDisplay}>
+                        ${renderAsCurrency(fiat.total.localized)}
+                    </span>
+                    &bull;
+                    <span style={valueDisplay}>
+                        ${renderAsCurrency(fiat.pair.localized)} ETC/{currentLocaleCurrency.toUpperCase()}
+                    </span>
+                </Col>
+            </Row>
         </div>
     );
 };
 
 Render.propTypes = {
     total: PropTypes.number.isRequired,
+    fiat: PropTypes.object.isRequired,
+    currentLocaleCurrency: PropTypes.string.isRequired,
 };
 
 
 const Total = connect(
-    (state, ownProps) => ({
-        total: state.accounts.get('accounts', Immutable.List()).map((account) => {
+    (state, ownProps) => {
+        // (whilei) I left hardcoded rates because they might be worth throwing in a popup, feed, sidebar, or
+        // some other kind of sexy sneaky UI down the road.
+
+        const oneEther = new Wei(ETHER);
+        const rates = state.accounts.get('rates');
+
+        // Sum of balances of all known accounts.
+        const total = state.accounts.get('accounts', Immutable.List()).map((account) => {
             if (account.get('balance')) {
                 return account.get('balance');
             }
             return new Wei(0);
-        }).reduce((t, v) => t.plus(v), new Wei(0)).getEther(),
-    }),
+        }).reduce((t, v) => t.plus(v), new Wei(0));
+        const totalEther = total.getEther();
+
+        let fiat = {};
+        let r = {};
+        if (rates && total) {
+            r = {
+                btc: rates.get('btc'),
+                eur: rates.get('eur'),
+                usd: rates.get('usd'),
+                cny: rates.get('cny'),
+            };
+            fiat = {
+                total: {
+                    btc: total.getFiat(r.btc),
+                    eur: total.getFiat(r.eur),
+                    usd: total.getFiat(r.usd),
+                    cny: total.getFiat(r.cny),
+                },
+                pair: {
```

Comment:
Thank you -- it was redundant to `r`; my reasoning was I wasn't totally our API exchange rates were going to be in Ether units (vs mwei, wei...) and thus had left the onus to `getFiat` to handle it. Have removed it.